install/macos.md

```diff
@@ -4,15 +4,18 @@ title: macOS-Installation
 ---
 
 
-Wir empfehlen die Installation einer 64-bit Version, falls man ein 64-bit Betriebssystem benutzt.
-Wenn man sich nicht sicher ist, hat man heutzutage fast immer 64-bit.
+Wir empfehlen die Installation einer 64-bit Version,
+wenn du ein 64-bit Betriebssystem benutzt.
+Wenn du dir nicht sicher bist, heutzutage ist fast jedes System 64-bit.
 
 __Es müssen alle Schritte (Installieren, Testen, Aktualisieren) ausgeführt werden!__
 
-Falls man nicht am LaTeX-Kurs teilnimmt, ist MacTeX optional.
-Dann kann man allerdings kein TeX in matplotlib benutzen (führt zu weniger schönen Plots).
+Falls du nicht am LaTeX-Kurs teilnimmst, ist MacTeX optional.
+Dann kannst du allerdings kein TeX in matplotlib benutzen (führt zu weniger schönen Plots).
 
-Falls man nur am LaTeX-Kurs teilnehmen will, sollte man mindestens VS Codium (oder einen anderen Texteditor) und MacTeX installieren.
+Falls du nur am LaTeX-Kurs teilnehmen willst,
+solltest du mindestens VS Codium (oder einen anderen Texteditor)
```

Comment:
As mentioned before in my comment on the Linux instructions: "VS Codium" vs "Visual Studio Codium"

Reply:
I changed it to VSCodium, as per their naming.